src/material/tabs/tab-group.ts

```diff
@@ -147,6 +148,10 @@ export class MatTabGroup implements AfterContentInit, AfterContentChecked, OnDes
   @Input({alias: 'mat-stretch-tabs', transform: booleanAttribute})
   stretchTabs: boolean = true;
 
+  /** Alignment for tabs label. */
+  @Input({alias: 'mat-align-tabs'})
+  alignTabs: 'start' | 'center' | 'end' | null = null;
```

Comment:
While this is the correct type, it looks like we'll have to set it to `string | null` since it breaks some cases internally.

Reply:
I updated it but internal cases CI is still pending not sure if recent push triggered it or not. 

![image](https://github.com/user-attachments/assets/ae290154-466c-4f1a-84ef-54df01d86af5)

also this is wrong place to ask but when I run `yarn` after fork it keeps failing with:

```
[3/5] Fetching packages...
error Couldn't find match for "36946be4df61f6549ae3829c026022e47674eae2" in "refs/heads/main" for "https://github.com/angular/dev-infra-private-ng-dev-builds.git".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
my-pc@mypc-Latitude-3310:~/Development/angular-projects/open-source/components$ 
```
I tried to manually install the package in a clean folder with only `@angular/ng-dev` package but it still failed I have a global setting for specific NPM packages but they only work with npm (not sure if they will collide with yarn) & also this is something that came up recently, I was able to install packages before IIRC.